docs/data/data-grid/overview/overview.md

```diff
@@ -43,6 +50,20 @@ import { DataGridPro } from '@mui/x-data-grid-pro';
 
 {{"demo": "DataGridProDemo.js", "defaultCodeOpen": false, "disableAd": true, "bg": "inline"}}
 
+#### Premium Plan [<span class="plan-premium"></span>](https://mui.com/store/items/mui-x-premium/)
+
+The Premium version contains all features from Pro, as well as exclusive features for data-rich applications.
```

Comment:
This would precision what we have in mind with "exclusive features", this is a more precise common denominators between all the Premium features, I believe:

```suggestion
The Premium version contains all features from Pro, as well as features for analyzing large datasets.
```